src/ios/CDVVisit.m

```diff
@@ -3,7 +3,7 @@
 #import "CDVVisit.h"
 #import <Cordova/CDV.h>
 
-@import CoreLocation;
```

Comment:
This should be the "proper" way of importing CoreLocation as you don't need to add the framework in the project settings. However, Xcode will complain if you do not enable 'Modules', disabled by default in this project (I guess we should blame Cordova for that).
Link: http://stackoverflow.com/questions/18947516/import-vs-import-ios-7